contracts/facets/DFMoveFacet.sol

```diff
@@ -24,10 +25,11 @@ import {
     PlanetExtendedInfo2,
     PlanetEventMetadata,
     PlanetEventType,
-    Upgrade
+    Upgrade,
+    ArenaPlayerInfo
 } from "../DFTypes.sol";
 
-contract DFMoveFacet is WithStorage {
+contract DFMoveFacet is WithStorage, WithArenaStorage {
```

Comment:
I think we should put this in `DFArenaCoreFacet` for consistency and cut it into the Diamond. In this implementation, we're breaking the separation between DF Core facets and Arena facets. What do you think?

Reply:
im ok to edit this then